cmd/openshift-tests/e2e.go

```diff
@@ -1,18 +1,24 @@
 package main
 
 import (
+	"regexp"
 	"strings"
 	"time"
 
 	"github.com/openshift/origin/pkg/synthetictests"
 	"github.com/openshift/origin/pkg/test/ginkgo"
 	exutil "github.com/openshift/origin/test/extended/util"
 	"k8s.io/kubectl/pkg/util/templates"
+	"k8s.io/kubernetes/openshift-hack/e2e/annotate"
 
 	_ "github.com/openshift/origin/test/extended"
 	_ "github.com/openshift/origin/test/extended/util/annotate/generated"
 )
 
+var (
+	excludedTestsFilter = regexp.MustCompile(strings.Join(annotate.ExcludedTests, `|`))
```

Comment:
No, you should exclude things that are not in conformance.

Reply:
why?  my goal here is to run any test that has established itself as reliable, regardless of what suite/bucket it is in.

Reply:
because tests in conformance mean one thing, and tests not in conformance mean "specialized suites that may not play well with others"

Reply:
I thought "disruptive" and "serial" meant "may not play well with others"?

Reply:
anyway i've updated to only include conformance tests.